BigriverBookstore-api/BigriverBookstore-api.Server.ServiceModel/Configurations/ApiEntryPointConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using JsonApiFramework.ServiceModel.Configuration;
+
+namespace BigriverBookstore_api.ServiceModel.Configurations
+{
+    public class ApiEntryPointConfiguration : ResourceTypeBuilder<ApiEntryPoint>
```

Comment:
You should move this into the web service, your domain models don't actually care that you are going to represent them as json.